### PR TITLE
CLI arg doc changes

### PR DESCRIPTION
**Pull request by stephenroller**
Mostly beautification

### CODE REVIEW

File: docs/source/generate_cli.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def get_scripts():
 
 
 def escape_output(str):
-    return str.replace('`', '``')
+    return str
```
Comment (klshuster) [inline docs/source/generate_cli.py]: if we're just removing this then do we even need the function?

File: parlai/agents/fairseq/fairseq.py [hunk 2]
```diff
@@ -6,6 +6,18 @@
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
 
+"""
+ParlAI had limited support for using models from
```
Comment (klshuster) [inline parlai/agents/fairseq/fairseq.py]: nit: has*

File: parlai/scripts/eval_wordstat.py [hunk 3]
```diff
@@ -12,19 +12,23 @@
 statistic for any agent given the agent object (with corr. dict) and a
 sequence.
 
-Example:
-    python eval_wordstat.py -mf data/model -t convai2:self
 
-One can specify bins boundaries with argument -fb | --freq-bins 10,100,1000
+Additionally provides function get_word_stats can be used in other parts of
```
Comment (klshuster) [inline parlai/scripts/eval_wordstat.py]: nit: that can*

File: parlai/scripts/train_model.py [hunk 4]
```diff
@@ -5,26 +5,24 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
-"""Train a model.
+"""The standard way to train a model. After training, also computes validation
+and test error.
 
-After training, computes validation and test error.
+The user must provide a model (with ``--model``) and a task (with ``--task``).
```
Comment (klshuster) [inline parlai/scripts/train_model.py]: Note: a user can also provide a task to be used with `PytorchDataTeacher` via `-pyt`, or can alternatively provide a pytorch `Dataset` via `-pytd`

**Comment by stephenroller** [comment]
Addressed Kurt's comments.

**Pull request merged**